core/services/functions/external_adapter_client.go

```diff
@@ -190,7 +194,10 @@ func (ea *externalAdapterClient) request(
 	req.Header.Set("Content-Type", "application/json")
 
 	start := time.Now()
-	client := &http.Client{}
+	retryClient := retryablehttp.NewClient()
+	retryClient.RetryMax = ea.maxRetry
```

Comment:
Is there a config value for the base delay between each retry attempt? I know it is exponential backoff, but I would think there is some base number they use to calculate the delay between each attempt. It would be nice if this value was also configurable from the jobspec.

Reply:
👍🏼  added a `exponentialBackoffBaseSec` parameter in which we can specify the base delay